### PR TITLE
feature/validating_placements_tests

### PR DESCRIPTION
**Pull request by JCL461437**
Adds tests for the valid_placement? method.

### CODE REVIEW

**Comment by JCL461437** [comment]
Great paired work!

File: spec/board_spec.rb [hunk 1]
```diff
@@ -16,16 +16,15 @@
             expect(@board).to be_an_instance_of(Board)
         end
         
-        it 'can collect all cells' do
+        it 'can  all cells' do
```
Comment (AustinKCodes) [inline spec/board_spec.rb:19]: did you mean to delete "collect" here?

Comment (JCL461437) [reply spec/board_spec.rb:19]: No, thanks for catching that man!

Comment (AustinKCodes) [reply spec/board_spec.rb:19]: Yea. I'll fix it when I pull it

**Pull request merged**